frontend/src/scenes/session-recordings/player/inspector/components/Timing/NetworkRequestTiming.tsx

```diff
@@ -40,6 +42,8 @@ const perfDescriptions: Record<(typeof perfSections)[number], string> = {
     'receiving response': 'The time taken to receive the response from the server.',
     'document processing':
         'The time taken to process the document after the response from the server has been received.',
+    server_timing:
```

Comment:
Not a blocker here but any reason this key is underscored but every other has a space?

Reply:
only laziness... the others are displayed to the user and this one isn't so I copied it over unchanged 😊

Reply:
will definitely be coming back to these components so will yolo for now